.github/workflows/ci-build.yaml

```diff
@@ -78,7 +78,7 @@ jobs:
   e2e-tests:
     name: E2E Tests
     runs-on: ubuntu-latest
-    timeout-minutes: 20
```

Comment:
Please revert the changes to build files. Those should go in another PR if needed.